exercises/04-integration/before/Divergent.ITOps/Divergent.ITOps.csproj

```diff
@@ -9,8 +9,8 @@
   <ItemGroup>
     <PackageReference Include="Autofac" Version="4.6.1" />
     <PackageReference Include="EntityFramework" Version="6.1.3" />
-    <PackageReference Include="NServiceBus" Version="7.0.1" />
```

Comment:
This change has not been made in the after project. Also, this change should be in a separate PR, since it's unrelated to the sqlp switch.